package.json

```diff
@@ -35,7 +35,7 @@
   ],
   "dependencies": {
     "can-cid": "^1.0.3",
-    "can-compute": "^3.3.1",
+    "can-compute": "^3.3.7",
```

Comment:
any reason we need to do this?

Reply:
`3.3.7` fixes https://github.com/canjs/can-compute/pull/110, which was breaking a test before I even started working on this.